linux/keyman-config/keyman_config/gsettings.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python3
+import logging
+import os
+import subprocess
+from gi.repository import Gio
+
+from gi.overrides.GLib import Variant
+
+
+class GSettings():
```

Comment:
This is lovely -- could you just add a comment on the front describing usage for Mr Posterity?

Reply:
done